.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: 'build-test'
+on: # rebuild any PRs and main branch changes
+  pull_request:
+  push:
+    branches:
+      - main
+      - 'releases/*'
+
+jobs:
+  build: # make sure build/ci work properly
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          npm install
+      - run: |
+          npm run all
+
+      - uses: ./
+        with:
+          github-token: ${{ github.token }}
```

Comment:
We usually access the github token like this: `${{ secrets.GITHUB_TOKEN }}`
I'm not sure if it'd be different in this context though

Reply:
thanks I've updated.